architecture/infrastructure_components/kubernetes_infrastructure.adoc

```diff
@@ -98,12 +102,16 @@ kubelet:
 - The kubelet listening for HTTP and responding to a simple API to submit a new
  manifest.
 
-=== Service Proxy [[service-proxy]]
+[[service-proxy]]
+
+=== Service Proxy
 
 Each node also runs a simple network proxy that reflects the services defined in
 the API on that node. This allows the node to do simple TCP and UDP stream
 forwarding across a set of back ends.
 
+[[node-object-specification]]
```

Comment:
Renamed this "Node Object Definition" to match other usage, and made the relevant link updates.